ibis/bigquery/tests/test_compiler.py

```diff
@@ -404,48 +404,58 @@ def test_projection_fusion_only_peeks_at_immediate_parent():
     assert result == expected
 
 
-def test_bool_reducers(alltypes):
+def test_bool_reducers(alltypes, project_id):
     b = alltypes.bool_col
     expr = b.mean()
     result = expr.compile()
     expected = """\
 SELECT avg(CAST(`bool_col` AS INT64)) AS `mean`
-FROM `ibis-gbq.testing.functional_alltypes`"""
+FROM `{}.testing.functional_alltypes`""".format(
+        project_id
+    )
     assert result == expected
 
     expr2 = b.sum()
     result = expr2.compile()
     expected = """\
 SELECT sum(CAST(`bool_col` AS INT64)) AS `sum`
-FROM `ibis-gbq.testing.functional_alltypes`"""
+FROM `{}.testing.functional_alltypes`""".format(
+        project_id
+    )
     assert result == expected
 
 
-def test_bool_reducers_where(alltypes):
+def test_bool_reducers_where(alltypes, project_id):
     b = alltypes.bool_col
     m = alltypes.month
     expr = b.mean(where=m > 6)
     result = expr.compile()
     expected = """\
 SELECT avg(CASE WHEN `month` > 6 THEN CAST(`bool_col` AS INT64) ELSE NULL END) AS `mean`
-FROM `ibis-gbq.testing.functional_alltypes`"""  # noqa: E501
+FROM `{}.testing.functional_alltypes`""".format(  # noqa: E501
+        project_id
+    )
     assert result == expected
 
     expr2 = b.sum(where=((m > 6) & (m < 10)))
     result = expr2.compile()
     expected = """\
 SELECT sum(CASE WHEN (`month` > 6) AND (`month` < 10) THEN CAST(`bool_col` AS INT64) ELSE NULL END) AS `sum`
-FROM `ibis-gbq.testing.functional_alltypes`"""  # noqa: E501
+FROM `{}.testing.functional_alltypes`""".format(  # noqa: E501
+        project_id
+    )
     assert result == expected
 
 
-def test_approx_nunique(alltypes):
+def test_approx_nunique(alltypes, project_id):
     d = alltypes.double_col
     expr = d.approx_nunique()
     result = expr.compile()
     expected = """\
 SELECT APPROX_COUNT_DISTINCT(`double_col`) AS `approx_nunique`
-FROM `ibis-gbq.testing.functional_alltypes`"""
+FROM `{}.testing.functional_alltypes`""".format(
```

Comment:
can u use f strings in these?

Reply:
Done in c775a05e6fe8c6d9538eb196df5628a6fc211657